documentation/specs/Spinner.md

```diff
@@ -0,0 +1,118 @@
+# `Spinner` Component Specification
```

Comment:
today it is filled to a fixed percentage and then rotates, but good point in that in the future it might be determinate. maybe it'd be worth using this:

https://react-spectrum.adobe.com/react-aria/useProgressBar.html

i notice they give a circle example and the spectrum progress circle uses that as an underlying primitive

Reply:
I'll add this during implementation